recipes/common/cosmic-scripts.yml

```diff
@@ -0,0 +1,11 @@
+type: script
+scripts:
+  - authselect.sh
+  - setfilepermissions.sh
+  - disablesealertpopups.sh
+  - homebrewanalyticsoptout.sh
```

Comment:
i was asking you to keep a common-scripts.yml so we don't have duplication

now we have two files that are almost entirely duplicated :)

please remove the duplication